### ISSUE TITLE
show-config command error

### ISSUE THREAD

**Issue by grundb** [repobee/repobee#159]
While running repobee with a config file, everything seems to work fine except for the `show-config` command which throws a critical error. Below is the output from running `verify-settings` and `show-config` in that order: 

```
$ repobee verify-settings 
[INFO] verifying settings ...
[INFO] trying to fetch user information ...
[INFO] SUCCESS: found user grundb, user exists and base url looks okay
[INFO] verifying oauth scopes ...
[INFO] SUCCESS: oauth scopes look okay
[INFO] trying to fetch organization inda-18 ...
[INFO] SUCCESS: found organization inda-18
[INFO] verifying that user grundb is an owner of organization inda-18
[INFO] SUCCESS: user grundb is an owner of organization inda-18
[INFO] GREAT SUCCESS: All settings check out!
```
```
$ repobee show-config
[ERROR] 'Namespace' object has no attribute 'github_base_url'
[INFO] exception was raised before pre-initialization was complete. This is usually due to incorrect settings. Try running the `verify-settings` command and see if the problem can be resolved. If all fails, please open an issue at https://github.com/repobee/repobee/issues/new and supply the stack trace below.
[ERROR] critical exception
Traceback (most recent call last):
  File "/home/johan/.local/lib/python3.6/site-packages/repobee/main.py", line 63, in main
    parsed_args, api = cli.parse_args(app_args)
  File "/home/johan/.local/lib/python3.6/site-packages/repobee/cli.py", line 116, in parse_args
    _validate_tls_url(args.github_base_url)
AttributeError: 'Namespace' object has no attribute 'github_base_url'
```

I have a config file from which it seems that `github_base_url` is fetched properly, as running comands such as clone works fine without supplying it as an argument.

**Comment by slarse** [comment]
Thanks for reporting it! This is a bug caused by some doofus ordering of operations in parsing CLI arguments, and is fixed upstream (see #156). Since a user actually was hit by it I guess it's time to create a new release :).

I'll notify in this issue once the new version which fixes the problem is available.

**Comment by slarse** [comment]
@grundb Fixed in version 1.2.1, upgrade and it should be fine. Close this issue once you've verified that it works :)
